src/native-common/Button.tsx

```diff
@@ -110,6 +134,12 @@ export class Button extends RX.Button<{}> {
         }
     }
 
+    public setNativeProps(nativeProps: RN.ViewProps) {
```

Comment:
No need to explicitly mark it public. It is assumed to be so. We don't use public elsewhere in ReactXP.